tools/python-scripts/get_input_py3.py

```diff
@@ -0,0 +1,251 @@
+#!/Users/heike/anaconda/bin/python
```

Comment:
this might not work for most people ...

Reply:
;)

Reply:
Oh no! :)

Reply:
Adapted. Thanks for the feedback Lukas!

Reply:
/bin/python also doesn't work everywhere.  Default debian for example is /usr/bin/python and even then one might want to use a different python.  The most robust solution I know of is '/usr/bin/env python'  but I don't know if that works for example on osx.

I didn't read all of your code, someone who knows more than me should do that ...

Reply:
I think somebody using the scripts should know enough python to adapt them. Thanks for the suggestions but I prefer not to change the first line again.

Reply:
Since no merge happened I changed the header. At the end it is best to be as general as possible.